tests/test_rules.py

```diff
@@ -27,6 +28,10 @@ def test_init_rule(self) -> None:
                         key='x', operator='=', value='2')
         self.assertEqual(actual, expected)
 
+    def test_header_whitespace_allowed(self) -> None:
+        # issue 33
```

Comment:
As much as possible can we try to not point to an issue on GitHub? It helps to have less back and forth.

Reply:
sure, we can rely on git-blame instead.